CODEOWNERS

```diff
@@ -0,0 +1,53 @@
+# Remember that the last applicable rule for any given file is the only one
+# that applies (if I am reading the docs correctly).
+
+# Default rule: @diyessi gets everything.
+*                                   @diyessi
+
+/cmake/                             @rkimballn1 @silee2
+
+/.ci/                               @crlishka
```

Comment:
We may need to split `.ci` into three parts. I can be responsible for `travis` and `onnx` subdirs

Reply:
@postrational @crlishka What do you guys think about:

```
/.ci/                               @crlishka
/.ci/travis                         @postrational
/.ci/onnx                           @postrational
```